### PR TITLE
change: [M3-7337] - change Linode Details Summary VPC IPv4 Text to Copy Object.

### PR DESCRIPTION
**Pull request by hasyed-akamai**
## Description 📝
We've changed the label from "VPC IPv4 Text" to "VPC IPv4 Copy Object" to make it clearer. Now, it's easier for users to find the option to copy the IPv4 object.

## Changes  🔄
List any change relevant to the reviewer.

- In Linode Details Summary Page:
  - changed the label from "VPC IPv4 Text" to "VPC IPv4 Copy Object"

## Preview 📷

| Before | After |
| ------ | ----- |
| Before | <video src="https://github.com/user-attachments/assets/98f370c2-0281-4abb-913c-41534ca377b5"></video> |

### Prerequisites
- You should have at least one Linode with VPC in your account.

### Reproduction steps
- Go to Linode Details Summary Page:
  - Observe VPC IPv4 Label as a Copy Object.

### Verification steps

- After changes, observe VPC IPv4 Label as a Copy Object.

## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

**Review by pmakode-akamai — COMMENTED** [review]
Thanks for the change!

✅ Linode Details Summary VPC IPv4 Text to Copy Object looks good in both light and dark modes and copy functionality work as expected.

❓ Do we need to change `Subnets` text to `Subnet`?
Screenshot 2024-10-29 at 2 33 20 PM



Unrelated to this PR, it looks like this is an existing styling issue. I wonder if this could be fixed as well?
Screenshot 2024-10-29 at 2 22 06 PM

File: packages/manager/src/features/Linodes/LinodeEntityDetailBody.tsx [hunk 1]
```diff
@@ -251,21 +260,31 @@ export const LinodeEntityDetailBody = React.memo((props: BodyProps) => {
...
-            </StyledVPCBox>
+            <StyledVPCWrapper>
+              <StyledVPCLabel data-testid="vpc-ipv4">VPC IPv4</StyledVPCLabel>
+              <StyledVPCItem component="span" data-testid="vpc-ipv4">
+                {configInterfaceWithVPC?.ipv4?.vpc && (
+                  <StyledTableCell sx={{ border: 0 }}>
+                    <StyledGradientDiv>
+                      <CopyTooltip
+                        copyableText
+                        text={configInterfaceWithVPC?.ipv4?.vpc}
```
[the remark this answers is not in the document]
Comment (hasyed-akamai) [reply packages/manager/src/features/Linodes/LinodeEntityDetailBody.tsx]: Make all the changes mentioned in the Comment.

**Review by pmakode-akamai — APPROVED** [review]

File: packages/manager/src/features/Linodes/LinodeEntityDetailBody.tsx [hunk 2]
```diff
@@ -251,21 +260,29 @@ export const LinodeEntityDetailBody = React.memo((props: BodyProps) => {
...
-                  VPC IPv4:
-                </StyledLabelBox>{' '}
-                {configInterfaceWithVPC?.ipv4?.vpc}
-              </StyledListItem>
-            </StyledVPCBox>
+            <StyledVPCWrapper>
+              <StyledVPCLabel data-testid="vpc-ipv4">VPC IPv4</StyledVPCLabel>
+              <StyledVPCItem component="span" data-testid="vpc-ipv4">
+                {configInterfaceWithVPC?.ipv4?.vpc && (
+                  <StyledTableCell sx={{ border: 0 }}>
```
Comment (harsh-akamai) [inline packages/manager/src/features/Linodes/LinodeEntityDetailBody.tsx:274]: Could we merge the styles of both `StyledVPCItem` and `StyledTableCell` into one styled component?

File: packages/manager/src/features/Linodes/LinodeEntityDetailBody.tsx [hunk 3]
```diff
@@ -251,21 +260,29 @@ export const LinodeEntityDetailBody = React.memo((props: BodyProps) => {
...
-                <StyledLabelBox component="span" data-testid="vpc-ipv4">
-                  VPC IPv4:
-                </StyledLabelBox>{' '}
-                {configInterfaceWithVPC?.ipv4?.vpc}
-              </StyledListItem>
-            </StyledVPCBox>
+            <StyledVPCWrapper>
+              <StyledVPCLabel data-testid="vpc-ipv4">VPC IPv4</StyledVPCLabel>
+              <StyledVPCItem component="span" data-testid="vpc-ipv4">
+                {configInterfaceWithVPC?.ipv4?.vpc && (
```
Comment (harsh-akamai) [inline packages/manager/src/features/Linodes/LinodeEntityDetailBody.tsx:273]: Lets add this check for the complete wrapper component

**Comment by hasyed-akamai** [comment]
@harsh-akamai Added all the changes you mentioned in the comments.

**Review by harsh-akamai — APPROVED** [review]
Changes look good! 🎉  Thanks Haneef

**Comment by abailly-akamai** [comment]
@hasyed-akamai and reviewers @harsh-akamai & @pmakode-akamai 
this PR introduced a pretty bad bug, see https://github.com/linode/manager/pull/11289

In the future, if you don't see a reviewer from our team assigned to one of your PRs, please assign one - thx!

**Comment by pmakode-akamai** [comment]
> @hasyed-akamai and reviewers @harsh-akamai & @pmakode-akamai this PR introduced a pretty bad bug, see #11289
> 
> In the future, if you don't see a reviewer from our team assigned to one of your PRs, please assign one - thx!

@abailly-akamai I saw your comment and have checked what led to the regressions.

I didn't notice any such regressions when I initially reviewed the PR. However, I was OOO when @harsh-akamai suggested additional styling changes, and unfortunately, these changes didn't go through my review before the PR was merged. Apologies for the inconvenience caused.

**Pull request merged**